Gemfile.lock

```diff
@@ -1,33 +1,41 @@
 PATH
   remote: .
   specs:
-    signature (0.1.8)
+    signature (1.0.0)
```

Comment:
Cleanup: Gemfile.lock didn't get updated after you changed the version number from Dylan's comment. You'll need to run `bundle lock` again and push up the change.